[WIP] Updating Angular versions

subhahu123 opened the pull request:
<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->
Fixes #990 #1428 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link
<!-- Replace "PR_NUMBER" with your pull request number. This link is generated when your PR passes the travis tests.A sample link can look like https://pr-200-fossasia-susper.surge.sh -->

#### Changes proposed in this pull request:

<!-- Changes: Add here what changes were made in this issue and if possible provide links. -->
-

<!-- Demo Link: Add here the link where you changes can be seen. -->

- https://pr-1429-fossasia-susper.surge.sh

<!-- Screenshots for the change: Add here the screenshot of the fix. -->
-

neeraj3029 reviewed (commented):
Please fix the travis build faliure

subhahu123 committed:
Updating Angular versions
Update app.module.ts
Update app.component.spec.ts
Update search.service.ts
Update search.service.spec.ts
Update news.service.ts
Update knowledgeapi.service.ts
Update knowledgeapi.service.spec.ts
Update intelligence.service.ts
Update intelligence.service.spec.ts
Update get-json.service.ts
Update autocorrect.service.ts
Update autocomplete.service.spec.ts
Update about.component.spec.ts
Update advancedsearch.component.spec.ts
Update auto-complete.component.spec.ts
Update auto-correct.component.spec.ts
Update contact.component.spec.ts
Update index.component.spec.ts
Update infobox.component.spec.ts
Update intelligence.component.spec.ts
Update navbar.component.spec.ts
Update not-found.component.spec.ts
Update results.component.spec.ts
Update search-bar.component.spec.ts
Update searchsettings.component.spec.ts
Update statsbox.component.spec.ts
Removing @angular/http
Update knowledgeapi.service.ts
Update knowledgeapi.service.spec.ts
Update get-json.service.ts
Update autocorrect.service.ts
Update autocomplete.service.spec.ts
Update autocomplete.service.spec.ts
Update searchsettings.component.spec.ts
Update search-bar.component.spec.ts
Update search-bar.component.spec.ts
Update results.component.spec.ts
Update results.component.spec.ts
Update not-found.component.spec.ts
Update not-found.component.spec.ts
Update navbar.component.spec.ts
Update intelligence.component.spec.ts
Update infobox.component.spec.ts
Update index.component.spec.ts
Update auto-correct.component.spec.ts
Update auto-complete.component.spec.ts
Update app.module.ts
Update advancedsearch.component.spec.ts
Update autocorrect.service.ts
Update statsbox.component.spec.ts
Update search.service.ts
Update advancedsearch.component.spec.ts
Update app.component.spec.ts
Update app.module.ts
Update auto-complete.component.spec.ts
Update results.component.spec.ts
Update not-found.component.spec.ts
Update navbar.component.spec.ts
Update intelligence.component.spec.ts
Update infobox.component.spec.ts
Update index.component.spec.ts
Update auto-correct.component.spec.ts
Update search-bar.component.spec.ts
Update searchsettings.component.spec.ts
Update autocomplete.service.spec.ts
Update intelligence.service.spec.ts
